cosmosResponse.go

```diff
@@ -9,21 +9,36 @@ import (
 	"github.com/supplyon/gremcos/interfaces"
 )
 
+//nolint:structcheck // reports responseStatusCode wrong
+type cosmosStatusCodeDescription struct {
+	responseStatusCode            int
+	retry                         bool
+	retryOnNewConnection          bool
```

Comment:
can you pls add a hint what `retryOnNewConnection` means / what it is intended for?